components/server/src/memory.rs

```diff
@@ -13,15 +13,32 @@ pub struct MemoryTraceManager {
 
 impl MemoryTraceManager {
     pub fn flush(&mut self, _now: Instant) {
+        let raft_router = "raft_router".to_string();
```

Comment:
Better to use `CONST` on these two strings.

Reply:
Good catch. They're all removed with a more general way.